{
  "_from": "react-transform-catch-errors@^1.0.2",
  "_id": "react-transform-catch-errors@1.0.2",
  "_inBundle": false,
  "_integrity": "sha1-G01KdulycYlvwW/jCGx5PsiKnus=",
  "_location": "/react-transform-catch-errors",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "react-transform-catch-errors@^1.0.2",
    "name": "react-transform-catch-errors",
    "escapedName": "react-transform-catch-errors",
    "rawSpec": "^1.0.2",
    "saveSpec": null,
    "fetchSpec": "^1.0.2"
  },
  "_requiredBy": [
    "/babel-preset-react-hmre"
  ],
  "_resolved": "https://registry.npmjs.org/react-transform-catch-errors/-/react-transform-catch-errors-1.0.2.tgz",
  "_shasum": "1b4d4a76e97271896fc16fe3086c793ec88a9eeb",
  "_spec": "react-transform-catch-errors@^1.0.2",
  "_where": "/Users/mmarinich/WebstormProjects 16.03.39/alice2.0/node_modules/babel-preset-react-hmre",
  "author": {
    "name": "Dan Abramov",
    "email": "dan.abramov@me.com"
  },
  "bugs": {
    "url": "https://github.com/gaearon/react-transform-catch-errors/issues"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "React Transform that catches errors inside React components",
  "devDependencies": {
    "babel-cli": "^6.3.17",
    "babel-core": "^6.3.21",
    "babel-preset-es2015": "^6.3.3",
    "babel-preset-stage-0": "^6.3.13",
    "rimraf": "^2.4.3"
  },
  "files": [
    "lib",
    "src"
  ],
  "homepage": "https://github.com/gaearon/react-transform-catch-errors#readme",
  "keywords": [
    "react-transform",
    "react",
    "reactjs",
    "errors",
    "rhl",
    "dx"
  ],
  "license": "MIT",
  "main": "lib/index.js",
  "name": "react-transform-catch-errors",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/gaearon/react-transform-catch-errors.git"
  },
  "scripts": {
    "build": "babel src --out-dir lib",
    "clean": "rimraf lib",
    "prepublish": "npm run clean && npm run build"
  },
  "version": "1.0.2"
}
